packages/config/index.js

```diff
@@ -29,6 +29,8 @@ function applyDefaultConfig(config) {
       locations: [],
       basePath: '',
       assetPath: '',
+      workerPath: '<assetPath>/sw.js',
```

Comment:
please replace `<assetPath>` with `<basePath>` to make sure the SW can control the actual app